packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -481,6 +481,11 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
       return backendId as UnknownIdentifier;
     });
 
+    if (idsToAdd !== []) {
+      // if we have id's to call then we want to set this state in the adapter to true.
+      this.getState().isPSTNorACSCall = true;
```

Comment:
When are we clearing this state variable?

Reply:
as long as the composite isn't going back to the home page this should be set so that the rejoin button does not appear. setting this state back to false would be to contoso's implementation of their app.

Reply:
If we have a PSTN participant in the call, but that participant leave in the middle of the call, do we still consider this call as a PSTN call?